### PR TITLE
update KolNovel

### PR DESCRIPTION
**Pull request by OtakuArab**

### CODE REVIEW

File: scripts/multisrc/lightnovelwp/sources.json [hunk 1]
```diff
@@ -10,10 +10,11 @@
   },
   {
     "id": "kolnovel",
-    "sourceSite": "https://kolnovel.club/",
+    "sourceSite": "https://kolbook.xyz/",
     "sourceName": "Kol Novel",
     "options": {
       "lang": "Arabic",
+      "versionIncrements": 1
```
Comment (error7404) [inline scripts/multisrc/lightnovelwp/sources.json:17]: You should put it to 2, has detailed in the readme

**Comment by error7404** [comment]
Duplicate of #1013

**Comment by nyagami** [comment]
image

File: scripts/multisrc/lightnovelwp/sources.json [hunk 2]
```diff
@@ -10,10 +10,11 @@
   },
   {
     "id": "kolnovel",
-    "sourceSite": "https://kolnovel.club/",
+    "sourceSite": "https://kolbook.xyz/",
     "sourceName": "Kol Novel",
     "options": {
       "lang": "Arabic",
+      "versionIncrements": 2
```
Comment (error7404) [inline scripts/multisrc/lightnovelwp/sources.json:17]: you forgot a "," at the end of this line

**Review by nyagami — APPROVED** [review]

**Pull request merged**